### PR TITLE
TFKUBE-306: Add confluence_enable_synchrony to docs

### PR DESCRIPTION
**Pull request by nanux**
Almost all the work was done by @errcode1202 in a previous PR, kudos! 🙇

### CODE REVIEW

File: docs/docs/userguide/CONFIGURATION.md [hunk 1]
```diff
@@ -338,7 +338,15 @@ desired_capacity = <NUMBER OF NODES>  # between 1 and 10
     confluence_min_heap            = "256m"
     confluence_max_heap            = "512m"
     ```
+    
+    ### Collaborative editing
+
+    `confluence_enable_synchrony` enables [Collaborative editing](https://confluence.atlassian.com/doc/collaborative-editing-858771779.html). (default: `true`)
 
+    ```terraform
+    confluence_enable_synchrony = true
```
Comment (nanux) [inline docs/docs/userguide/CONFIGURATION.md:347]: When I was adding the property, shouldn't we call it `confluence_enabled_collaborative_editing`? I am not sure if Synchrony is a commonly known term.

Comment (errcode1202) [reply docs/docs/userguide/CONFIGURATION.md:347]: I agree I think `confluence_enabled_collaborative_editing` is a better fit

**Comment by nanux** [comment]
@errcode1202 the shortcoming of using the tabs for the product configuration is that if you have selected a specific product (e.g. Bamboo) and you will click on the property from a different product (e.g. Confluence License) in the right sidebar, it doesn't scroll to the property. You need to select the correct tab to scroll to the right place.
Screen Shot 2022-02-15 at 5 13 37 pm

**Comment by errcode1202** [comment]
> @errcode1202 the shortcoming of using the tabs for the product configuration is that if you have selected a specific product (e.g. Bamboo) and you will click on the property from a different product (e.g. Confluence License) in the right sidebar, it doesn't scroll to the property. You need to select the correct tab to scroll to the right place. Screen Shot 2022-02-15 at 5 13 37 pm

@nanux thats annoying I guess the `pymdownx.tabbed` extension doesn't support that behaviour. Do you think there is a tradeoff we need to make here between aesthetics and navigation?

**Comment by errcode1202** [comment]
@nanux to address this Ive created this PR https://github.com/atlassian-labs/data-center-terraform/pull/158 as discussed

File: docs/docs/userguide/CONFIGURATION.md (hunk 1 above)
Comment (nghazali) [reply docs/docs/userguide/CONFIGURATION.md:347]: I think even `confluence_collaborative-editing_enabled` is better option.

**Review by nghazali — APPROVED** [review]

**Review by errcode1202 — APPROVED** [review]

File: docs/docs/userguide/CONFIGURATION.md (hunk 1 above)
Comment (nanux) [reply docs/docs/userguide/CONFIGURATION.md]: I have renamed the property to `confluence_collaborative_editing_enabled`.

**Pull request merged**